### PR TITLE
replace my old vue-draggable-nested-tree with he-tree-vue; add my new vue-colrow

### PR DESCRIPTION
**Pull request by phphe**
he-tree-vue is a new version of vue-draggable-nested-tree. vue-colrow is to help layouting UI and responsive layout.

### CODE REVIEW

**Comment by phphe** [comment]
- [x] git: link not to repository
- [x] git: no conflicts resolved

The link is to docs site.

**Comment by phphe** [comment]
> @phphe Could you update this PR if it is relevant. In any case, let me know if something is necessary.
> 
> ## Checklist
> * [x]  git: link not to repository
> * [ ]  git: broken PR
> * [ ]  git: no conflicts resolved
> * [ ]  time: not updated for more than 6 months
> * [ ]  category: not in its category
> * [ ]  lists: not the last on the list
> * [ ]  docs: no documentation
> * [ ]  docs: non-english documentation
> * [ ]  project: dead project
> * [ ]  project: does not accept donat
> * [ ]  site: no link to the site
> * [ ]  site: site with errors
> * [ ]  site: not actively using vuejs
> * [ ]  site: not original
> 
> > more about [CONTRIBUTING](https://github.com/vuejs/awesome-vue/blob/master/CONTRIBUTING.md)

The link is to docs site.

**Comment by phphe** [comment]
> @phphe Could you update this PR if it is relevant. In any case, let me know if something is necessary.
> 
> ## Checklist
> * [x]  git: link not to repository
> * [ ]  git: broken PR
> * [ ]  git: no conflicts resolved
> * [ ]  time: not updated for more than 6 months
> * [ ]  category: not in its category
> * [x]  lists: not the last on the list
> * [ ]  docs: no documentation
> * [ ]  docs: non-english documentation
> * [ ]  project: dead project
> * [ ]  project: does not accept donat
> * [ ]  site: no link to the site
> * [ ]  site: site with errors
> * [ ]  site: not actively using vuejs
> * [ ]  site: not original
> 
> > more about [CONTRIBUTING](https://github.com/vuejs/awesome-vue/blob/master/CONTRIBUTING.md)

The link is to docs site directly. he-tree-vue is new version of my vue-draggable-nested-tree, so I replaced it with he-tree-vue, so he-tree-vue, is not last on the list.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,4 @@
-<p align="center">
-  <br>
-  <img width="400" src="./assets/logo.svg" alt="logo of vue-awesome repository">
-  <br>
-  <br>
-</p>
+
```
Comment (Scrum) [inline README.md]: @phphe please revert this change

File: README.md [hunk 2]
```diff
@@ -1468,7 +1463,7 @@ Tooltips / popovers
  - [Bosket](https://github.com/elbywan/bosket) - Collection of tree view components for front-end frameworks (Vue, React, Angular and Riot).
  - [plantain-00/tree-component](https://github.com/plantain-00/tree-component) - A reactjs, angular and vuejs tree component.
  - [sl-vue-tree](https://github.com/holiber/sl-vue-tree) - A simple customizable draggable tree component for Vue.js
- - [vue-draggable-nested-tree](https://github.com/phphe/vue-draggable-nested-tree) - A powerful customizable draggable tree view component for Vuejs2 [@phphe](https://github.com/phphe)
+ - [he-tree-vue](https://he-tree-vue.phphe.com) - A draggable sortable vue tree component, with dragging placeholder, types definition. 14 KB gzipped. Under MIT license.
```
Comment (Scrum) [inline README.md]: @phphe  link not to repository

File: README.md [hunk 3]
```diff
@@ -1724,7 +1719,7 @@ Tooltips / popovers
  - [vue-drag-it-dude](https://github.com/Esvalirion/vue-drag-it-dude) - Vue2 component, that allows you to drag object wherever you want.
  - [vue-draggable](https://github.com/Vivify-Ideas/vue-draggable) - Vue Drag and Drop library without any dependency. Simple and easy to use.
  - [vue-nestable](https://github.com/rhwilr/vue-nestable) - A simple drag & drop hierarchical list made as a vue component.
- - [vue-draggable-nested-tree](https://github.com/phphe/vue-draggable-nested-tree) - A powerful customizable draggable tree view component for Vuejs2 [@phphe](https://github.com/phphe)
+ - [he-tree-vue](https://he-tree-vue.phphe.com) - A draggable sortable vue tree component, with dragging placeholder, types definition. 14 KB gzipped. Under MIT license.
```
Comment (Scrum) [inline README.md]: @phphe doublicate

File: README.md [hunk 4]
```diff
@@ -1981,6 +1976,7 @@ Tooltips / popovers
  - [vue-mock-layout](https://github.com/promosis/vue-mock-layout) - Easily mock the layout of your Vue apps.
  - [vue-simple-drawer](https://github.com/dreambo8563/vue-simple-drawer) - A tiny drawer panel with bounced animation, nest supported and theme customized. directions: left/right/up/down
  - [vue-grd](https://github.com/1000ch/vue-grd) - Simple, Light-weight and Flexible Vue.js component for grid layout.
+ - [vue-colrow](https://vue-colrow.phphe.com) - Smarter layout components: Row, Col. Based on css flexbox. Responsive layout. Support SSR, use fixed, percentage and grow width together. Suitable for form layout, cards list. 5 KB gzipped.
```
Comment (Scrum) [inline README.md]: @phphe not the last on the list

**Comment by phphe** [comment]
Sorry I will recreate 2 pull requests.

File: README.md [hunk 5]
```diff
@@ -2160,6 +2156,7 @@ composing CSS breakpoint state.
  - [fine-mq](https://github.com/nash403/fine-mq) - A fine API to manage media queries in JS with ease and first-class integration with VueJS as a plugin.
  - [vue-responsive-components](https://github.com/Kelin2025/vue-responsive-components) - Create responsive components with `ResizeObserver`.
  - [vue-screen-size](https://github.com/promosis/vue-screen-size) - Get easy and reactive access to the width and height of your screen.
+ - [vue-colrow](https://vue-colrow.phphe.com) - Smarter layout components: Row, Col. Based on css flexbox. Responsive layout. Support SSR, use fixed, percentage and grow width together. Suitable for form layout, cards list. 5 KB gzipped.
```
Comment (Scrum) [inline README.md]: @phphe doublicate
